sandbox.go

```diff
@@ -104,6 +104,7 @@ type resolvConfPathConfig struct {
 type containerConfig struct {
 	hostsPathConfig
 	resolvConfPathConfig
+	name              string
```

Comment:
Unless we need the sandbox to store the container name for something else, it should not be needed to fix this issue.